WordPress/Classes/Models/Comment+CoreDataClass.swift

```diff
@@ -106,6 +106,17 @@ public class Comment: NSManagedObject {
     @objc func isTopLevelComment() -> Bool {
         return depth == 0
     }
+
+    func isFromPostAuthor() -> Bool {
+        guard let post = post,
+              let postAuthorID = post.authorID?.int32Value,
```

Comment:
Nitpick - this could just be `guard let postAuthorID = post?.authorID?.int32Value`.

Reply:
Thank you! I initially planned to check other properties (e.g. author name) in case if the `authorID` does not exist or has invalid values. But ended up returning `false` in case that results in a false positive.

This should be addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/17589/commits/ce1d2c7c7083cdf55b54bf8bef19f002845cef5f.